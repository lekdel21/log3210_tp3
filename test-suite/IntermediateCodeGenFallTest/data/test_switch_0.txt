num b;
enum A {
    B, C, D
}
A a;

a = B;

switch(a) {
    case B:
        b = 5;
        break;
}