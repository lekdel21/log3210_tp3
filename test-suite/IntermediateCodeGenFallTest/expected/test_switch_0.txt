a = 0
_L1
if a == 0 goto _L2
goto _L0
_L2
b = 5
goto _L0
_L0
